docs/retrying-test.adoc

```diff
@@ -25,6 +25,12 @@ The test function will be executed, up to `maxAttempts`, until it completes succ
 
 The default for `minSuccess` is 1. The value must be greater than or equal to 1.
 
+=== suspendForMs [optional]
+
+The `suspendForMs` attribute specifies the amount of time to pause between retries, in milliseconds.
+
+The default for `suspendForMs` is 0 (no pause). The value must be greater than or equal to 0.
```

Comment:
Can you add a sentence along the lines of "The thread executing this test is sleeping during that time and won't execute other tests, so long suspensions are discouraged."? Thanks!

Reply:
Also, can you add an example and describe what happens in it?